packages/create-itp-react-app/templates/.sketchxport/postprocess.sh

```diff
@@ -2,7 +2,8 @@
 set -e
 
 function main() {
-  echo "This file is picked up by sketchxport-worker".
-  echo "You can perform additional transforms & tasks on output here."
+  # This file is picked up by sketchxport-worker
+  # You can perform additional transforms & tasks on output here
+  node ./.sketchxport/postprocess.js
```

Comment:
Could be that this should be relative to sketchxport folder since the worker will control where the script cwd is.

Reply:
ok, change to ./postprocess.js then?

Reply:
Lets test out of this works and otherwise create a bug?